tc-tests-utils.sh

```diff
@@ -1110,6 +1132,17 @@ do_deepspeech_nodejs_build()
       clean node-wrapper
   done;
 
+  for electron in ${SUPPORTED_ELECTRONJS_VERSIONS}; do
+    EXTRA_CFLAGS="${EXTRA_LOCAL_CFLAGS}" EXTRA_LDFLAGS="${EXTRA_LOCAL_LDFLAGS}" EXTRA_LIBS="${EXTRA_LOCAL_LIBS}" make -C native_client/javascript \
+      TARGET=${SYSTEM_TARGET} \
+      RASPBIAN=${SYSTEM_RASPBIAN} \
+      TFDIR=${DS_TFDIR} \
+      NODE_ABI_TARGET=--target=$electron \
+      NODE_DIST_URL=--disturl=https://atom.io/download/electron \
+      NODE_RUNTIME=--runtime=electron \
+      clean node-wrapper
+  done;
+
```

Comment:
Is this roughly doubling the size of the Node package as well as the time to build it? Could we separate Node.JS and Electron packages, or does that not make sense in the Node ecosystem?

Reply:
It seems that it does not make sense in the ecosystem, but it's not really doubling, we just add a few `deepspeech.node` binaries, ranging from 50k to 100k. All docs I found would pull from `npmjs` for ElectronJS. And build time does increase, but it's marginal, takes < 30 secs for each ElectronJS instance. We would win much more build time by doing the heavy lifting of producing ready-to-use Docker images.

Reply:
Understood. My review+ remains :)